control-plane-operator/hostedclusterconfigoperator/cmd.go

```diff
@@ -16,8 +16,12 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/openshift/hypershift/support/util"
```

Comment:
Nit: Importgrouping, please do not put third party imports into the stdlib group

Reply:
done